entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+if [[ -z "$*" ]]; then
+  echo "Expecting one or more arguments"
+  exit 1
+fi
+
+#remove spack if exist previously
+if [[   -d "$GITHUB_WORKSPACE"/temp/spack ]]; then
```

Comment:
we can expose an optional environment variable to delete the folder only if users request it. Also, let's install to `$GITHUB_WORKSPACE/install` by default (as we had it before), but also create a variable for this.